README.md

```diff
@@ -48,42 +48,83 @@ Both GCC 4.8 and Clang 3.4 meet this requirement.
 To install on a Debian or Ubuntu system, first install the prerequisites:
 
 ```sh
-sudo apt-get install autoconf automake libtool make g++ pkg-config libboost-dev \
+sudo apt-get install make cmake g++ libboost-dev lua5.2 liblua5.2-dev \
   libboost-system-dev libboost-filesystem-dev libboost-thread-dev libexpat1-dev \
-  libgeos-dev libgeos++-dev libpq-dev libbz2-dev libproj-dev zlib1g-dev \
-  lua5.2 liblua5.2-dev
+  libgeos-dev libgeos++-dev libpq-dev libbz2-dev libproj-dev zlib1g-dev
 ```
 
 To install on a Fedora system, use
 
 ```sh
-sudo yum install gcc-c++ automake libtool pkgconfig boost-devel \
-  expat-devel bzip2-devel postgresql-devel geos-devel proj-devel \
-  lua-devel
+sudo yum install gcc-c++ cmake boost-devel lua-devel expat-devel \
+  bzip2-devel postgresql-devel geos-devel proj-devel zlib-devel proj-epsg
+```
+
+On RedHat / CentOS some packages are places in EPEL repositories:
+
+```sh
+sudo yum install epel-release
+sudo yum install geos-devel proj-devel proj-epsg
+```
+
+If the default version of PostgreSQL is too old, the official RPM repository
```

Comment:
This is getting into how to install PostgreSQL, not building osm2pgsql. What version does Centos 7 have?